src/main/java/kitchenpos/tablegroup/application/OrderTableValidator.java

```diff
@@ -0,0 +1,79 @@
+package kitchenpos.tablegroup.application;
+
+import kitchenpos.common.ErrorCode;
+import kitchenpos.menu.domain.Menu;
+import kitchenpos.menu.domain.MenuRepository;
+import kitchenpos.order.application.OrderValidator;
+import kitchenpos.order.domain.Order;
+import kitchenpos.order.domain.OrderRepository;
+import kitchenpos.tablegroup.domain.OrderTable;
+import kitchenpos.tablegroup.domain.OrderTableRepository;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.List;
+
+@Component
+public class OrderTableValidator implements OrderValidator {
+    private static final int NUMBER_OF_MINIMUM_ORDER_MENU = 1;
+
+    private final MenuRepository menuRepository;
+    private final OrderRepository orderRepository;
+    private final OrderTableRepository orderTableRepository;
+
+    public OrderTableValidator(MenuRepository menuRepository, OrderRepository orderRepository,
+                          OrderTableRepository orderTableRepository) {
+        this.menuRepository = menuRepository;
+        this.orderRepository = orderRepository;
+        this.orderTableRepository = orderTableRepository;
+    }
+
+    @Override
+    public void validateToCreateOrder(Long orderTableId, List<Long> menuIds) {
+        OrderTable orderTable = findOrderTableById(orderTableId);
+        List<Menu> menus = findAllMenuById(menuIds);
+        if (orderTable.isEmpty()) {
+            throw new IllegalArgumentException(ErrorCode.ORDER_TABLES_CANNOT_BE_EMPTY.getErrorMessage());
+        }
+        if (menus.size() < NUMBER_OF_MINIMUM_ORDER_MENU) {
```

Comment:
```suggestion
        if (menus.미만감증(NUMBER_OF_MINIMUM_ORDER_MENU)) {
```

가져와서 비교하기보다는 전달해볼수있겠네요 😄